modules/cache_builder/config/cache_builder.php

```diff
@@ -248,19 +248,27 @@
     from taxon_lists tl
     join taxa_taxon_lists ttl on ttl.taxon_list_id=tl.id
     #join_needs_update#
-    join taxa_taxon_lists ttlpref
+    -- Select the preferred name which isn't redundant (allow_data_entry=true)
+    left join taxa_taxon_lists ttlpref
       on ttlpref.taxon_meaning_id=ttl.taxon_meaning_id
-      and ttlpref.preferred=true
+      and ttlpref.preferred='t'
```

Comment:
I'd prefer to see a consistent use of either 't' or true for a boolean, rather than a mix. https://www.postgresql.org/docs/9.1/datatype-boolean.html states that true (or TRUE) is preferred